docs/common/header.md

```diff
@@ -1,29 +1,12 @@
 <link rel="stylesheet" href="{{baseUrl}}/css/main.css">
 <navbar placement="top" type="inverse">
   <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand"><img src="{{baseUrl}}/images/logo-darkbackground.png" height="20" /></a>
-  <dropdown text="User Guide" class="nav-link">
-    <li><a href="{{baseUrl}}/userGuide/index.html" class="dropdown-item">Home</a></li>
-    <li><a href="{{baseUrl}}/userGuide/userQuickStart.html" class="dropdown-item">Quick Start</a></li>
-    <li role="separator" class="dropdown-divider"></li>
-    <li><a href="{{baseUrl}}/userGuide/developingASite.html" class="dropdown-item">Developing a Site</a>
-    <li><a href="{{baseUrl}}/userGuide/siteConfiguration.html" class="dropdown-item">◦&nbsp; Site Configuration</a></li>
-    <li role="separator" class="dropdown-divider"></li>
-    <li><a href="{{baseUrl}}/userGuide/contentAuthoring.html" class="dropdown-item">Content Authoring</a></li>
-    <li><a href="{{baseUrl}}/userGuide/includingContents.html" class="dropdown-item">◦&nbsp; Including Contents</a></li>
-    <li><a href="{{baseUrl}}/userGuide/usingComponents.html" class="dropdown-item">◦&nbsp; Using Components</a></li>
-    <li><a href="{{baseUrl}}/userGuide/usingVariables.html" class="dropdown-item">◦&nbsp; Using Variables</a></li>
-    <li><a href="{{baseUrl}}/userGuide/pageLayout.html" class="dropdown-item">◦&nbsp; Page Layout</a></li>
-    <li><a href="{{baseUrl}}/userGuide/knownProblems.html" class="dropdown-item">◦&nbsp; Known Problems</a></li>
-    <li role="separator" class="dropdown-divider"></li>
-    <li class="dropdown-header">Deploying a Site</li>
-    <li><a href="{{baseUrl}}/userGuide/ghpagesDeployment.html" class="dropdown-item">◦&nbsp; Github Pages Deployment</a></li>
-    <li><a href="{{baseUrl}}/userGuide/netlifyPreview.html" class="dropdown-item">◦&nbsp; Preview site on Netlify</a></li>
-    <li role="separator" class="dropdown-divider"></li>
-    <li><a href="{{baseUrl}}/userGuide/components.html" class="dropdown-item">Components Reference</a></li>
-  </dropdown>
+  <li><a href="{{baseUrl}}/index.html" class="nav-link">HOME</a></li>
+  <li><a href="{{baseUrl}}/userGuide/index.html" class="nav-link">DOCS</a></li>
+  <li><a href="{{baseUrl}}/showcase.html" class="nav-link">SHOWCASE</a></li>
+  <li><a href="{{baseUrl}}/about.html" class="nav-link">ABOUT</a></li>
```

Comment:
@damithc @yamgent It seems the removed dropdown means we can no longer navigate to those pages on mobile.

Reply:
That is indeed a problem. Ideally, the SiteNav should automatically transform into a drop-down menu. What do you guys think?

Reply:
Created #591.